### PR TITLE
Add contribution policy and moderation section to edit collective

### PR DESCRIPTION
**Pull request by sbinlondon**
Relates https://github.com/opencollective/opencollective-api/pull/4640
Relates https://github.com/opencollective/opencollective/issues/3466

Updated screenshot:

Screenshot 2020-09-30 at 14 49 30

### CODE REVIEW

File: components/edit-collective/Menu.js [hunk 1]
```diff
@@ -69,9 +70,9 @@ const SECTION_LABELS = defineMessages({
     id: 'editCollective.menu.export',
     defaultMessage: 'Export',
   },
-  [EDIT_COLLECTIVE_SECTIONS.EXPENSES]: {
-    id: 'editCollective.menu.expenses',
-    defaultMessage: 'Expenses Policy',
+  [EDIT_COLLECTIVE_SECTIONS.POLICIES]: {
+    id: 'editCollective.menu.policies',
+    defaultMessage: 'Collective Policies',
```
Comment (Betree) [inline components/edit-collective/Menu.js:75]: Given that this section is also displayed on funds, and might one day be displayed on events, maybe we can simply call it `Policies`?

**Review by Betree — APPROVED** [review]
Looking good!

**Comment by znarf** [comment]
Have you considered having the "Rejected Categories" part of the Policies section?

I think we already have a lot of entries in the Menu and don't see the need for a separate section at this point.

**Comment by znarf** [comment]
I wrote that to explain the categorization system, can we add it near the form:

> All categorized Financial Contributors are manually classified by the Open Collective team.

> Only contributors that are thought to be abusing are classified with these categories. Financial Contributors with a good reputation should normally not be affected by this setting. 

Also, here is an update to the category system:

```
  ADULT: 'Adult websites',
  AFFILIATE_REVIEW: 'Affiliate and Review websites',
  CASINO_GAMBLING: 'Casino and Gambling',
  VPN_PROXY: 'VPN and Proxy services',
  WRITER: 'Essay writing services',
  SEO_AUDIENCE: 'SEO services & Social Media buying',
```

**Comment by znarf** [comment]
Instead of using the word "filter" (as in `filteredContributionCategories`) I recommend to use the word "reject" which describes better what we're doing here.

**Comment by sbinlondon** [comment]
> Also, here is an update to the category system:
> 
> ```
>   ADULT: 'Adult websites',
>   AFFILIATE_REVIEW: 'Affiliate and Review websites',
>   CASINO_GAMBLING: 'Casino and Gambling',
>   VPN_PROXY: 'VPN and Proxy services',
>   WRITER: 'Essay writing services',
>   SEO_AUDIENCE: 'SEO services & Social Media buying',
> ```

Why are we removing crypto?

**Comment by znarf** [comment]
@sbinlondon Is there a way to avoid the duplicate save button?

**Comment by sbinlondon** [comment]
> @sbinlondon Is there a way to avoid the duplicate save button?

Sure, can work on that

File: components/edit-collective/Menu.js [hunk 2]
```diff
@@ -167,7 +167,7 @@ const sectionsDisplayConditions = {
   [EDIT_COLLECTIVE_SECTIONS.CONNECTED_ACCOUNTS]: c => isHost(c) || (isCollective(c) && !isFund(c)),
   [EDIT_COLLECTIVE_SECTIONS.UPDATES]: c => isFeatureAllowed(c, FEATURES.UPDATES),
   [EDIT_COLLECTIVE_SECTIONS.CONVERSATIONS]: c => isFeatureAllowed(c, FEATURES.CONVERSATIONS) && !isFund(c),
-  [EDIT_COLLECTIVE_SECTIONS.EXPENSES]: c => isCollective(c) || isFund(c),
+  [EDIT_COLLECTIVE_SECTIONS.POLICIES]: c => isCollective(c) || isFund(c) || isHost(c),
```
Comment (znarf) [inline components/edit-collective/Menu.js:170]: Why did you hast Hosts here @sbinlondon ?

Comment (sbinlondon) [reply components/edit-collective/Menu.js:170]: I did not realise the expenses policy was a separate section in the hosts menu area. Wil remove.

**Pull request merged**